ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -79,9 +79,25 @@ then
 
 	# Internal I2C bus
 	hmc5883 -C -T -I -R 4 start
-
+	
 	# Internal SPI bus ICM-20608-G
 	mpu6000 -T 20608 start
+	
+	# External UART DMA bus AMOV-IMU
+	if param compare SYS_COMPANION 1500000
```

Comment:
This should be in the main SYS_COMPANION section.

Reply:
But agar the configuration is not all the same for different autopilots

Reply:
It's starting a serial device connected to a special serial port that's handled consistently across boards. Why would this be any different?

Reply:
I mean there will be not only one external IMU so we really need more ports but they actually aren't consistent in various versions